### PR TITLE
Adding command pattern

### PR DESCRIPTION
**Pull request by nikhalster**

### CODE REVIEW

**Comment by sagarr** [comment]
Can you make filename all smallcase keeping it consistent with other files.

File: design-patterns/14_Command.ipynb [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## The Command Pattern encapsulates a request as an object, thereby letting you parameterize other objects\n",
+    "## with different requests, queue or log requests, and support undoable operations.\n",
+    "\n",
+    "![title](Pictures/command_design_pattern.png)\n"
```
Comment (sagarr) [inline design-patterns/14_Command.ipynb]: Also, not sure how you gonna embed your local pictures to notebook?

Comment (rvpanchani) [reply design-patterns/14_Command.ipynb]: you can use link locations from actual pictures

**Comment by nikhalster** [comment]
updated img links with imggur urls and renamed the files. kindly review

**Pull request merged**